recipes/ocean_wave_tracing/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "ocean-wave-tracing" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocean_wave_tracing-{{ version }}.tar.gz
+  sha256: 54a06d09a0b57683686405ba15dbf51aed03a85ebd05a20321e2060bb1fae073
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python =>3.7
+    - pip
+    - poetry
+  run:
+    - python >=3.7
+    - matplotlib-base
+    - numpy >=1.17
+    - scipy >=1.2
+    - netcdf4
+    - xarray
+    - pyproj >=2.3
+    - cartopy
+    - pip
+    - yaml
+    - ipython
+    - cmocean
+    - jupyterlab
+    - pytest
+    - simplejson
+
+test:
+  imports:
+    - simplejson
+    - simplejson.tests
```

Comment:
Test this package; not simplejson.

Reply:
Thanks for pointing this out. Now removed